apps/mdn/mdn-aws/infra/main.tf

```diff
@@ -131,11 +130,6 @@ module "efs-us-west-2-prod" {
   efs_name    = "prod"
   vpc_id      = data.terraform_remote_state.vpc-us-west-2.outputs.vpc_id
   subnets     = data.terraform_remote_state.vpc-us-west-2.outputs.public_subnets
-
-  nodes_security_group = flatten([
-    data.aws_security_groups.us-west-2-nodes_sg.ids,
-    data.terraform_remote_state.eks-us-west-2.outputs.mdn_apps_a_worker_security_group_id,
```

Comment:
Out of curiosity: why are you removing here security groups, including the ones on the new eks nodes?

Reply:
Yeah I changed the way the security groups work here, EKS managed nodes doesn't expose security groups so I can't add a source security group anymore.

Instead I whitelisted based on CIDRs